src/Quote.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { memo, forwardRef, ReactNode } from "react";
+import { symToStr } from "tsafe/symToStr";
+import { assert } from "tsafe/assert";
+import type { Equals } from "tsafe";
+import { fr } from "./lib";
+import { cx } from "./lib/tools/cx";
+
+// We make users import dsfr.css, so we don't need to import the scoped CSS
+// but in the future if we have a complete component coverage it
+// we could stop requiring users to import the hole CSS and only import on a
+// per component basis.
+import "./dsfr/component/quote/quote.css";
+import { FrClassName } from "./lib/generatedFromCss/classNames";
+
+export type QuoteProps = {
+    className?: string;
+    text: string;
+    author?: string;
+    source?: ReactNode;
+    sourceUrl?: string;
+    image?: string;
+    size?: "md" | "lg" | "xl";
```

Comment:
```suggestion
    size?: "md" | "lg" | "xl";
```
Could you use more explicit terms.  
I think `md` `lg` `xl` is confusing, it's usually breakpoint keys.  
[Example in `<Button />`](https://github.com/codegouvfr/react-dsfr/blob/2fdff992c711edeeec8ce14087e26f3a17adb8bb/src/Button.tsx#L24-L25)  

Also could you add JSDoc to specify the default that we use.  [Example](https://github.com/codegouvfr/react-dsfr/blob/2fdff992c711edeeec8ce14087e26f3a17adb8bb/src/Button.tsx#L24)

Reply:
Ha ok, ça vient de la [page de démo du Quote](https://main--ds-gouv.netlify.app/example/component/quote/)

mais ok je fais comme sur bouton c'est plus explicite #consistency

Reply:
Oui tu as raison, c'est évident comme choix....